espnet2/asr/encoder/transformer_encoder.py

```diff
@@ -15,7 +16,7 @@
 from espnet.nets.pytorch_backend.transformer.multi_layer_conv import Conv1dLinear
 from espnet.nets.pytorch_backend.transformer.multi_layer_conv import MultiLayeredConv1d
 from espnet.nets.pytorch_backend.transformer.positionwise_feed_forward import (
-    PositionwiseFeedForward,
+    PositionwiseFeedForward,  # noqa: H301
```

Comment:
This comma, `PositionwiseFeedForward,` is added by Black.
Black style allows multiple import and they say that even one-import statement should end with comma if parenthesis is used.

https://github.com/psf/black/issues/1220

but it conflicts with hacking-extension. `hacking` sends error if it ends with comma.

I agree almost that Black developer said, and  I also think this hacking rule is reasonable for me.
Thus, finally, I decided to exclude them one by one.